src/assets/translations/en/main.json

```
@@ -155,6 +155,7 @@
     "symmetric": "Symmetric",
     "asymmetric": "Asymmetric (%{assetSymbol})",
     "import": "Import",
+    "loadingAccounts": "Loading Accounts",
```

suggestion: possibly nicer copy for users
```suggestion
    "accountsLoading": "Accounts are currently loading",
```

---

Might be too big for a button, resulting in a 2 lines buttons, specially on mobile, wdyt?

---

I tested, looks ok, let's try this one and change later if needed, updated in [2d57d12](https://github.com/shapeshift/web/pull/7629/commits/2d57d12de291dbebf2c14ad97f3338aff5d7b571)